.circleci/config.yml

```diff
@@ -118,6 +145,17 @@ workflows:
           filters:
             tags:
               only: /^v([0-9])+.([0-9])+.([0-9])+.*/
+      - publish-check:
```

Comment:
Should we also add `publish-check` as a requirement for `publish-stable`?

Reply:
publish-stable calls the verify command as well. We could add it as a dependency but not sure if it is worth it spinning up a new job/container and add an extra minute to the job.